cdap-docs/admin-manual/source/installation/cloudera.rst

```diff
@@ -543,6 +543,45 @@ For Kerberos-enabled Hadoop clusters:
 
   Restart CDAP after removing the usercache(s).
 
+
+.. _cloudera-configuration-integration-sentry:
+
+Enabling Sentry
+---------------
+To use CDAP with Cloudera clusters using Sentry authorization, these properties need to be set:
+
+- Add the user ``cdap`` to the Hive property ``sentry.metastore.service.users``
+- Add the user ``cdap`` to the Sentry property ``sentry.service.admin.group``
+- Add the user ``cdap`` to the Sentry property ``sentry.service.allow.connect``
+
+The Hive properties are set in ``hive-site.xml``, while the Sentry properties are set in
+``sentry-site.xml``.
+
+These can be set from within Cloudera Manager by using the appropriate "safety valve":
+
+- For Hive, use the *Hive Service Advanced Configuration Snippet (Safety Valve) for
+  hive-site.xml* (API ``hive_service_config_safety_valve``), setting it as::
+
+    <property>
+        <name>sentry.metastore.service.users</name>
+        <value>hive,impala,hue,hdfs,cdap</value>
+    </property>
+  
+- For Sentry, use the *Hive Service Advanced Configuration Snippet (Safety Valve) for 
+  sentry-site.xml*  (API ``hive_server2_sentry_safety_valve``), setting it as::
+  
+    <property>
+        <name>sentry.service.admin.group</name>
+        <value>hive,impala,hue,cdap</value>
+    </property>
```

Comment:
actually, we will still recommend this setting. This will not bypass authorization, but this allows the CDAP user to create roles, add roles to users, remove roles, list all roles, etc.

In fact, Apache Kafka "expects" that "kafka" is added in this setting. The sentry extension wouldn't work.

So, long story short, we require that the user provide us one of the groups listed here via a config setting. This setting will default to 'cdap'. So its not required, but recommended.